lib/AnswerHash.pm

```diff
@@ -115,7 +115,7 @@ The answer hash class is guaranteed to contain the following instance variables:
 =cut
 
 BEGIN {
-	be_strict(); # an alias for use strict.  This means that all global variable must contain main:: as a prefix.
+#	main::be_strict(); # an alias for use strict.  This means that all global variable must contain main:: as a prefix.
     
```

Comment:
I'm just trying to figure out this change.  If I uncomment this I get warnings about not being able to find `main::be_strict()`.  If I change it back to the way it was I dont get any warnings, but I dont know whats different.